assets/src/edit-story/units/dimensions.js

```diff
@@ -0,0 +1,39 @@
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_WIDTH, PAGE_HEIGHT } from '../constants';
+
+export function dataUnit( v ) {
+	return Number( v.toFixed( 0 ) );
+}
+
+export function editorUnit( v ) {
+	return Number( v.toFixed( 5 ) );
+}
+
+export function dataToEditorX( x, pageWidth ) {
```

Comment:
It would be good to have some short function comments here.

Reply:
Done.